runtime.go

```diff
@@ -504,6 +504,8 @@ func (r *Runtime) init() {
 		accessor:     true,
 		configurable: true,
 	})
+
+	r.SetRateLimiter(rate.NewLimiter(rate.Inf, maxInt))
```

Comment:
@Gabri3l I adjusted this slightly, if I set burst to 1 it was always hitting the fillBucket logic so it was throwing off my benchmarks a bit.